.github/workflows/docs_test.yml

```diff
@@ -0,0 +1,54 @@
+name: Docs_test
+
+on: [push, pull_request]
+
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.9]
+
+    steps:
+    - name: apt-get stuff needed for libstell and vmec
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y build-essential gfortran openmpi-bin libopenmpi-dev libnetcdf-dev libnetcdff-dev liblapack-dev libscalapack-mpi-dev libhdf5-dev libhdf5-serial-dev git m4 libfftw3-dev libboost-all-dev libopenblas-dev
+
+    - uses: actions/checkout@v3
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: which pip after python setup
+      run: python -m pip install --upgrade pip
+
+    - name: Install sphinx and its dependencies for building docs
+      run: |
+        pip install sphinx sphinx-rtd-theme sphinxcontrib-napoleon sphinx-autodoc-napoleon-typehints
```

Comment:
Would it make sense to use `pip install -r docs/requirements.txt` here, to provide some test coverage for the `docs/requirements.txt` file?